pyproject.toml

```diff
@@ -94,12 +99,14 @@ bitbucket_dc_token = "awx_plugins.credentials.plugins:bitbucket_dc_token"
 gce = "awx_plugins.credentials.plugins:gce"
 azure_rm = "awx_plugins.credentials.plugins:azure_rm"
 github_token = "awx_plugins.credentials.plugins:github_token"
+github_app_token = "awx_plugins.credentials.github_app:github_app_token"
```

Comment:
@chrismeyersfsu does this really need both a regular credential plugin and a managed credential plugin?

Reply:
UPD: @arrestle confirmed that the managed credential type isn't needed.